src/sensors.py

```diff
@@ -0,0 +1,114 @@
+import time
+import psutil
+import requests
+
+
+def handle_coral_dev_board_temp():
```

Comment:
I renamed the file to metrics.py - https://github.com/collabora/MLBench/blob/main/metrics.py

Reply:
I was using the previous copy. Renamed sensors to metrics now. but I am using `metrics.py` from inside `src/` I have removed this one.